src/widgets/StepperInput.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TextInput, View, TouchableOpacity, StyleSheet } from 'react-native';
+
+import IonIcon from 'react-native-vector-icons/Ionicons';
+import { SUSSOL_ORANGE, APP_FONT_FAMILY } from '../globalStyles/index';
+
+/**
+ * Stepper component to update a numerical value. The `value` passed will be
```

Comment:
:nit The first word should be renamed StepperInput to match the change in name

Reply:
d765425 changed up the comment, hopefully it's clear!